src/app/plan/duck/reducers.ts

```diff
@@ -341,6 +345,117 @@ export const planCloseAndDeleteFailure = (state = INITIAL_STATE, action) => {
   return unlockPlan(state, action.planName);
 };
 
+/* 
+Hook Reducers
+*/
+
+export const removeHookRequest =
+  (state = INITIAL_STATE, action: ReturnType<typeof PlanActions.removeHookRequest>) => {
```

Comment:
I think these `ReturnType<...>` annotations are only useful if the types of the variables being returned in the referenced function are well-defined. It might be better to be explicit about all these types wherever we use `ReturnType`, but that's a much bigger change. We should at least try to put an explicit return type annotation on the action functions.

Reply:
+1 candidate for a global issue